docs/error-mapping-tchannel-http.md

```diff
@@ -0,0 +1,34 @@
+# Error Mapping between TChannel and HTTP
+
+## Stability: unstable
+
+[![unstable](http://badges.github.io/stability-badges/dist/unstable.svg)](http://github.com/badges/stability-badges)
+
+TChannel can return many different types of errors when making requests and
+sending responses.
+
+## TChannel Client Errors
+
+As part of http state machine integration, tchannel errors need be mapped to
+proper http status codes/messages so that http clients can differentiate errors
+and possibly take some actions.
+
+A list of tchannel errors defined at protocol level:
+[section payloads.error.code `protocol.md`](protocol.md)
+
+A list of http status code/message:
+[https://en.wikipedia.org/wiki/List_of_HTTP_status_codes]
+
+The proposed mapping is:
+
+| code   | name                 | http status code/message
+| ------ | -------------------- | -------------------------
+| `0x01` | timeout              | 408 Request Timeout
```

Comment:
Hmm 408 seems wrong, which means "the server timed out waiting for the request".  Probably a 504 gateway timeout makes more sense here.